qml/LeftPanel.qml

```diff
@@ -49,6 +49,25 @@ RowLayout {
         visible: (visiblePanel ===  panelItem[LeftPanel.LeftSubPanel.Status]) ? true : false
     }
 
+    ChangeAliasDialog {
+        id: aliasDialog
+    }
+
+    function changeAlias(entityId, currentAlias, entityKind) {
```

Comment:
Move both functions to the bottom of the file to be consistent with the other qml files

Reply:
:+1: